tests/markdown/check-markdown.test.ts

```diff
@@ -93,13 +99,14 @@ describe('Frontmatter Validator', () => {
 
     await expect(checkMarkdownFiles(invalidFolderPath, validateBlogs)).rejects.toThrow('ENOENT');
 
-    expect(logger.error.mock.calls[0][0]).toContain('Error in directory');
+    expect((logger.error as jest.Mock).mock.calls[0][0]).toContain('Error in directory');
```

Comment:
You can mock logger.error directly via jest.fn() or any other mock functions, instead of defining it explicitly as jest.Mock.